src/ui/public/field_editor/field_editor.html

```diff
@@ -16,7 +16,7 @@
       class="form-control"
       data-test-subj="editorFieldName">
   </div>
-  <div ng-if="editor.creating && editor.indexPattern.fields.byName[editor.field.name]" class="hintbox">
```

Comment:
Here we could replace `editor.indexPattern.fields.byName[editor.field.name]` with `editor.existingFieldNames.includes(editor.field.name)`